### PR TITLE
adds ci functionality

### PR DESCRIPTION
**Pull request by manne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+
+> A show case for [tapio](https://tapio.one/)
+
+## Build
+
+Execute the following commands
+
+```bash
+cd src\web
+ng build
```
Comment (SebastianRoether) [inline README.md]: npm install

Comment (SebastianRoether) [reply README.md]: And what about the .\build.ps1?

File: build.fsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+open Fake.Core.TargetOperators
+open Fake.DotNet
+open Fake.JavaScript
+
+BuildServer.install [
+    TeamFoundation.Installer
+]
+
+Target.create "Install.Prerequisites" (fun _ ->
+    if not BuildServer.isLocalBuild then
```
Comment (SebastianRoether) [inline build.fsx]: Why only if not building locally?

File: README.md (hunk 1 above)
Comment (manne) [reply README.md]: I will update the readme in a later PR, there will be a script to install all the prerequisites

File: build.fsx (hunk 2 above)
Comment (manne) [reply build.fsx]: see above.
> I will update the readme in a later PR, there will be a script to install all the prerequisites

**Comment by SebastianRoether** [comment]
Please remove dist/ and adjust .gitignore

File: README.md [hunk 3]
```diff
@@ -1 +1,52 @@
...
+```
+
+Execute the following commands in another shell
+
+```bash
+cd src\web
+dotnet run
+```
+
+Navigate to <https://localhost:5001> with a browser
```
Comment (SebastianRoether) [inline README.md]: Can be any other port without launchSettings

**Comment by SebastianRoether** [comment]
The web.csproj looks a bit broken when opened in VS (and also crashes my VS...)

**Pull request merged**